dataclients/kubernetes/kube.go

```diff
@@ -491,6 +494,20 @@ func (c *Client) fetchDefaultFilterConfigs() defaultFilters {
 	return filters
 }
 
+func (c *Client) GetEndpointAdresses(ns, name string) []string {
+	if c.state == nil {
+		log.Info("state is nil")
+		return nil
+	}
+
+	addrs := c.state.GetEndpointsByTarget(ns, name, "TCP", &definitions.BackendPort{
+		Value: 6379,
```

Comment:
Why is this hardcoded? Should it just get endpoints from the pre-configured namespace by the pre-configured name?

Reply:
We can also pass this in a func but then we would need to also expose this as config, and maybe this is too much

Reply:
Can you check if this is correct? I think it matches port by name which is not provided here and works because there is a single port in redis endpoints

Reply:
The code works, I tested the original one in our dev cluster.
Basically it's a copy from routegroup: https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/routegroup.go#L257-L262
The clusterstate is a cache with the cache key `endpointID` as we pass

Reply:
I want to say that you can put any number instead of `6379` and it will work

Reply:
I think I am wrong and it matches the port by value.
Maybe we can have a third config `KubernetesRedisServicePort` with the default value of 6379, otherwise it would not be possible to run Redis on another port.